images/kubekins-e2e/variants.yaml

```diff
@@ -22,13 +22,13 @@ variants:
     KIND_VERSION: 0.10.0
   master:
     CONFIG: master
-    GO_VERSION: 1.19.5
+    GO_VERSION: 1.20
```

Comment:
this has to be quoted, like the other `1.2x` strings or else it parses as a number and the 0 gets dropped leaving go 1.2 😅 

https://github.com/kubernetes/test-infra/pull/28660

Reply:
😅